LICENSES/LicenseRef-Slint-commercial.md

```diff
@@ -1,6 +1,394 @@
-## The Slint Commercial License
+# **Slint Software License Agreement**
+Version 1.2, 22 August 2022
 
-Slint is offered under a commercial license. Please contact info@slint-ui.com if you're interested.
+Copyright © 2022 SixtyFPS GmbH <https://slint-ui.com/imprint.html>
 
-See also the [Licensing FAQ](https://github.com/slint-ui/slint/blob/master/FAQ.md#licensing) online
-or find the same information in `FAQ.md` in the top level of your slint repository checkout.
+
+SixtyFPS GmbH with offices at Am Panke-Park 47, 16321 Bernau
+bei Berlin, Germany (“**SixtyFPS**”) grants a license to the software 
+(“**Licensed Software**”) as specified in **Exhibit A** which is provided 
+subject to the terms of this Slint Software License Agreement (the “**Agreement**”). 
+Each Party to this Agreement may be referred to 
+herein individually as a “**Party**” or collectively as the “**Parties**.”
+
+This Agreement incorporates by reference the following Exhibits
+that are attached to this Agreement:
+
+**Exhibit A Licensed Software**
```

Comment:
Is this relevant to the license text? This feels like it belongs into an actual contract only.

Reply:
have reconstructed the Exhibits and removed the Support part as it can be considered separate from the license grant.